install/docker/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   meshery:
-    image: layer5/meshery:latest
+    image: layer5/meshery:stable-latest
```

Comment:
Looks like someone had changed this tag for meshery image. 🤔

As there is no image tag of latest for layer5/meshery image.  https://hub.docker.com/r/layer5/meshery/tags?page=1&name=latest